### ISSUE TITLE
[Feature Request]: Add a way to create (and keep in sync) a GIST from Quiver

### ISSUE THREAD

**Issue by acrodrig** [HappenApps/Quiver#383]
My current workflow involves creating a lot of small Markdown documents for other people to read. I want to keep them in Quiver, as it is a nice container and editor for them.

When I want to distribute those, I end up creating a gist and I do a copy/paste. If I need to update, I repeat said process. It would be very nice to have a Button/Menu Option that says:

`Create/Update Gist`

And then it magically happens (after entering GitHub credentials/token).

Another (nice) option is for Quiver to have [App Extensions](https://developer.apple.com/app-extensions/) that allow developer to scratch their own itch.

**Comment by JMichaelTX** [comment]
> It would be very nice to have a Button/Menu Option that says:
> 
> Create/Update Gist

:+1:    I also could make good use of this feature.

**Comment by langtianlang** [comment]
+1

**Comment by dkarter** [comment]
Would love this feature!

**Comment by dkarter** [comment]
There are a few suggestions around this feature (#706 #298). I suggest closing those, and consolidating the discussion to here.

Also I think the implementation of this feature can use SnippetLab as a template - they have a two way sync, but just exporting should be fairly easy using the github gist API.

**Comment by floooat** [comment]
+1

**Comment by ubergoober** [comment]
I agree. I was one of those that requested it from SnippetsLab, and he added it. But unfortunately development for that product seems to have stalled and the Gists integration seems a bit unfinished (at least the 2 way sync). Adding Gists support to Quiver would be awesome.
